docs/Images/Domain/CoordinateMaps/CylindricalFlatEndcap.svg

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I very much appreciate these figures, they make a world of difference in understanding the maps.